### DISCUSSION TITLE
2 x11kwh lifepo4 - different SOCs

### DISCUSSION THREAD

**Discussion by exenbema** [Louisvdw/dbus-serialbattery#295] [Q&A] [answered]
hello, i have 2x11kwh blocks with daly bms and its good up and running. but since the beginning (8 month) until now i watched differnt soc states...why is this and what can i do?

i post a pic that show that
poblem is, when one battery is full and this marks then "allow to charge" - NO --the second battery also stops charging (i think)

big thx to this forum! kind regards markus

**VenusOS (please complete the following information):**
 - latetest Firmware (okt 2022)

**Battery/BMS (please complete the following information):**
 - BMS/Battery type: [Daly]
 - Cells: [16 per block) 16x230AH
 - Interface: [ USB

![36207517-c153-4d5d-90d5-7cbcea9f5cba](https://user-images.githubusercontent.com/114774449/202897238-1973ba5b-81c0-4fd8-bdac-0df45806526f.jpg)

**Comment by Louisvdw** [comment, marked as answer]
Hi
All batteries experience SOC drift, where the estimate doesn’t match what is happening in the battery. The BMS would normally readjust the SOC when some tollerances are seen. Most BMS are calibrated to measure larger current values which means that smaller currents cannot be measured and then some drift happens. Also no 2 batteries or cells are exactly the same and power will move easier where there is the smallest resistance.

The one thing you should check is that the power cables are the same thickness for each battery and the same length from the battery to the inverter.

**Comment by exenbema** [comment]
hi, thats excactly my problem  - different lenghts of the cables - thx soo much!! thx

**Comment by janekx** [comment]
I have 3 banks, 105Ah + 105Ah + 135Ah 
cables diferences 70mm2 have only 50cm max so first bank 1m, secod bank 1.5m and third 2m. But longest wires have 135Ah bank what is very fast drained so lowes internal resistance. The diferences are so big that 50% SoC at 135Ah bank and rest says 85% SoC.  when take 30A in sum from batts it look like 5A+5A+20A. So most issue is internal resistance

**Reply by exenbema** [reply]
thx but if you have the same cells, the internal restistance should be equal or not?

**Reply by janekx** [reply]
not really, 105 are same and 135 is diferent brand so lower resistance then 105. In fact resistance change in time, so older cellas will have diferent resitance then new
